ludo/src/main/java/de/uniks/ludo/controller/IngameController.java

```diff
@@ -101,7 +101,7 @@ public void setupDice() {
     }
 
     @onKey(code = KeyCode.R)
-    private void rollDice() {
+    public void rollDice() {
```

Comment:
package-private is also fine and should be our best practice -- it is the lowest visibility that works.